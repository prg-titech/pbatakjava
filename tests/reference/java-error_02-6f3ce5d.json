{
    "basename": "java-error_02-6f3ce5d",
    "cmd": "{infile}",
    "infile": "tests/error_02/build.sh",
    "infile_hash": "1c1e941f0c07049ca0ac9eb4ca6dc4c2d5c63a85de844ff2cafa7636",
    "stdout": null,
    "stdout_hash": null,
    "stderr": "java-error_02-6f3ce5d.stderr",
    "stderr_hash": "55cf5939ee37bea1d497a29d429a40330812b63fbcac5cc03ea0f64e",
    "returncode": 4
}