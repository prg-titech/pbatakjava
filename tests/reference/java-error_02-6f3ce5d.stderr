tests/error_02/Test.java:4: error: no applicable constructor for new Line«1,1»(new Color(), new Point())
Solution not found.
